h/models/annotation_slim.py

```diff
@@ -96,5 +96,7 @@ class AnnotationSlim(Base):
     )
     group = sa.orm.relationship("Group")
 
+    meta = sa.orm.relationship("AnnotationMetadata", uselist=False)
```

Comment:
`metadata` is reserved by SQLA here, using just meta (we could go for `metadata_`)

Reply:
`meta` sounds fine. Maybe we could add a comment there explaining that `metadata` is a reserved keyword.